keystone.go

```diff
@@ -43,6 +46,43 @@ type KeystoneToken struct {
 	Issued_At string
 }
 
+type KeystoneTokenv3 struct {
```

Comment:
You don't need a full structure. You need only fields that you would use. So I think that only IssueAt and ExpiresAT

Reply:
Done